src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -55,11 +60,64 @@ export async function deployImageToAzure(context: IActionContext, node?: RemoteT
 
     const site = nonNullProp(wizardContext, 'site');
     const createdNewWebApp: string = `Successfully created web app "${site.name}": https://${site.defaultHostName}`;
+    if (node instanceof AzureTagTreeItem) {
+        let siteClient = new SiteClient(site, node.parent.parent.parent.root);
+        let appUri: string = (await siteClient.getWebAppPublishCredential()).scmUri;
+        await createWebhookForApp(node, wizardContext, appUri);
+    } else if (node instanceof DockerHubTagTreeItem) {
+        // point to dockerhub to create a webhook
+        // http://cloud.docker.com/repository/docker/<registryName>/<repoName>/webHooks
+        const dockerhubPrompt: string = "Copy web app endpoint and browse to dockerhub";
```

Comment:
Can you show a screenshot? This seems like a big button...

Reply:
Yes 

![image](https://user-images.githubusercontent.com/30420973/60471878-62f84a80-9c1b-11e9-96b0-a6c3eac504a2.png)

Any suggestions on the naming?

Reply:
I'd follow this pattern: 
![image](https://user-images.githubusercontent.com/5290572/61319131-6e22ad00-a7bb-11e9-8fe0-ee11458c8d54.png)

Reply:
Done